tools/codegen/gen_backend_stubs.py

```diff
@@ -256,7 +263,43 @@ def gen_dispatcher_registrations(
 
     assert class_name is not None
     backend_index = backend_indices[dispatch_key]
+
+    dispatch_registrations_body = list(concatMap(
+        dest.RegisterDispatchKey(
+            backend_index,
+            Target.REGISTRATION,
+            selector,
+            rocm=False,
+            cpp_namespace=cpp_namespace,
+            class_method_name=f'{class_name}'),
+        grouped_native_functions
+    ))
+    deferred_dispatch_registrations = ""
+    static_init_dispatch_registrations = ""
+    if eager_registration:
+        static_template = CodeTemplate("""\
+TORCH_LIBRARY_IMPL(aten, $dispatch_key, m) { 
+    $dispatch_registrations_body
+};""")
+        static_init_dispatch_registrations = static_template.substitute(
+            dispatch_key=dispatch_key,
+            dispatch_registrations_body=dispatch_registrations_body
+        )
+    else:
+        deferred_template = CodeTemplate("""\
```

Comment:
minor nit: most of our codegen uses f-strings now instead of `CodeTemplate`, e.g:
```
deferred_dispatch_registrations = f"""\
TORCH_API void Register{backend_name}{dispatch_key}NativeFunctions() {{
...
```

Reply:
is it bad to use CodeTemplate though? I started off using f-strings, but since I am substituting a multi-line string (with indentation) CodeTemplate actually provides value here.

Reply:
I think f-strings are pretty much superior to CodeTemplate (you can do multi-line f-strings just fine, like in the codegen [here](https://github.com/pytorch/pytorch/blob/d014772b9f97523156e8d316a37b5f7e646cd247/tools/codegen/dest/register_dispatch_key.py#L86)).  It's not too big of a deal though so i don't wanna get hung up on it.